### PR TITLE
[Docs] Inferencer docs

### PR DESCRIPTION
**Pull request by gaotongxiao**

### CODE REVIEW

**Comment by Harold-lkk** [comment]
1. maybe provide a sample cli for inference in https://mmocr.readthedocs.io/en/dev-1.x/get_started/quick_run.html#inference.
2. Maybe firstly introduce cli and infer.py args in the user guide and then task inferencer
3. Inferencer in user guide and basic concept have too many similar contents.

**Comment by Harold-lkk** [comment]
fix https://mmocr--1744.org.readthedocs.build/zh_CN/1744/get_started/install.html#id6

**Review by Harold-lkk — APPROVED** [review]

File: docs/zh_cn/user_guides/inference.md [hunk 1]
```diff
@@ -1,192 +1,531 @@
...
-MMOCR 为示例和应用，以 [ocr.py](/mmocr/ocr.py) 脚本形式，提供了方便使用的 API。
+在 OpenMMLab 中，所有的推理操作都被统一到了推理器 `Inferencer` 中。推理器被设计成为一个简洁易用的 API，它在不同的 OpenMMLab 库中都有着非常相似的接口。
 
-该 API 可以通过命令行执行，也可以在 python 脚本内调用。在该 API 里，MMOCR 里的所有模型能以独立模块的形式被调用或串联。
+MMOCR 中存在两种不同的推理器：
 
-```{warning}
-该脚本仍在重构过程中，在接下来的版本接口中有可能会发生变化。
-```
+- **标准推理器**：MMOCR 中的每个基本任务都有一个标准推理器，即 `TextDetInferencer`，`TextRecInferencer`，`TextSpottingInferencer` 和 `KIEInferencer`。它们具有非常相似的接口，具有标准的输入/输出协议，并且总体遵循 OpenMMLab 的设计。这些推理器也可以被串联在一起，以便对一系列任务进行推理。
```
Comment (gaotongxiao) [inline docs/zh_cn/user_guides/inference.md:7]: mark the corresponding task.

File: docs/zh_cn/stats.py [hunk 2]
```diff
@@ -89,16 +91,29 @@ def title2anchor(name):
...
-# 统计数据
+# 总览
+
+## 权重
+以下是可用于[推理](user_guides/inference.md)的权重列表。
+
+每个权重可能有多个名称，表格中将用“/”分隔。
+尽管它们本质上是等价的，但此处提供较短的名称以方便参考。例如，`DB_r18` 和 `dbnet_resnet18_fpnc_1200e_icdar2015` 指向的是同一个权重。
+
+{weight_list}
```
Comment (gaotongxiao) [inline docs/zh_cn/stats.py:106]: add examples

**Pull request merged**